client/components/jetpack/videopress-iframe/index.tsx

```diff
@@ -0,0 +1,44 @@
+import classnames from 'classnames';
+import './style.scss';
```

Comment:
[NITPICK] We usually keep one line space before the style import.

Reply:
Nitpick away; still learning the proper conventions, so this is helpful. Addressed: 4a193b7c60